src/cryptenroll/cryptenroll-fido2.c

```diff
@@ -13,7 +13,8 @@ int enroll_fido2(
                 size_t volume_key_size,
                 const char *device,
                 bool lock_with_pin,
-                bool lock_with_up) {
+                bool lock_with_up,
+                bool lock_with_uv) {
```

Comment:
I'd prefer if this three bools would be replaced via some new flags argument, so that this becomes `Fido2EnrollFlags lock_with` or so